### ISSUE TITLE
Questions about "Test net output #0: detection_eval = 0"

### ISSUE THREAD

**Issue by keson1984** [Xilinx/Vitis-AI-Tutorials#5]
Hello, 
I download the Vitis-AI-ssd/SSD and retrain the Mobilenetv2-SSD in the Vitis-AI-ssd/SSD/workspace/Mobilenetv2-SSD. the log as follow:

I0521 00:17:13.147367  1523 solver.cpp:772] Iteration 6000, Testing net (#0)
I0521 00:17:13.148303  1523 net.cpp:743] Ignoring source layer mbox_loss
I0521 00:19:13.689599  1523 solver.cpp:885]     Test net output #0: detection_eval = 0
I0521 00:19:19.519392  1523 solver.cpp:270] Iteration 6000 (0.149189 iter/s, 670.289s/100 iter), loss = 3.38898, remaining 212 hours and 14 minutes
I0521 00:19:19.519454  1523 solver.cpp:291]     Train net output #0: mbox_loss = 3.49844 (* 1 = 3.49844 loss)
I0521 00:19:19.519470  1523 sgd_solver.cpp:106] Iteration 6000, lr = 0.001
...........................................

why "Test net output #0: detection_eval = 0". it confuse me for almost all the day, I can't find any useful solution in google.
( I use caffe-xilinx 1.1 )

train.sh: 
/workspace/caffe-xilinx/build/tools/caffe train -solver="/workspace/Vitis-AI-ssd/SSD/workspace/Mobilenetv2-SSD/solver.prototxt" \
-weights="/workspace/Vitis-AI-ssd/SSD/workspace/Mobilenetv2-SSD/pretrained.caffemodel" -gpu 0,1,2,3  2>&1 | tee train.log:q

**Comment by luxiaojian77** [comment]
Hello. Did you solve it? I suffered the same problem.

**Comment by DrChungAlbert** [comment]
This tutorial, however, uses Caffe provided within Vitis AI 1.1.

**Comment by sunshine411** [comment]
@DrChungAlbert 
I use Vitis-AI-1.2/tree/master/AI-Model-Zoo/caffe-xilinx suffered the same problem Test net output #0: detection_eval = 0
also test on xilinx/vitis-ai:lastest get same problem.

I don't understand which version you are using to solve this problem。
Can you tell me more specific information？
